diagnostic.md

```diff
@@ -21,13 +21,19 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET http://example.com/donuts \
+
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = (donuts) => {
+  $.ajax({
+    method: 'GET',
+    url: 'http://example.com/donuts',
+  })
+}
```

Comment:
you passed in a parameter donuts but did not do anything with it. and you forgot the success and fail callbacks, otherwise the rest of the call looks good